setup.py

```diff
@@ -53,6 +53,14 @@ def run(self):
 ]
 docker = ['docker-py>=1.6.0']
 druid = ['pydruid>=0.2.1']
+gcloud = [
+    'gcloud>=1.1.0',
+]
+gcp_api = [
+    'httplib2',
+    'google-api-python-client',
```

Comment:
@criccomini I think that for compatibility this might be better as `google-api-python-client<1.5` but I"m not sure. As far as I can tell 1.5 is the first version that depends on `oauth2client 2.0`.